GitUI/UserControls/BlameControl.cs

```diff
@@ -265,6 +270,10 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
 
             GitBlameCommit lastCommit = null;
 
+            var dateTimeFormat = AppSettings.BlameHideAuthorTime
+                ? CultureInfo.CurrentCulture.DateTimeFormat.ShortDatePattern
+                : CultureInfo.CurrentCulture.DateTimeFormat.ShortDatePattern + " " + CultureInfo.CurrentCulture.DateTimeFormat.ShortTimePattern;
```

Comment:
We should also honour user's date presentation settings (_Settings > Appearance > General_)
![image](https://user-images.githubusercontent.com/4403806/56948186-b80fd580-6b72-11e9-97ac-726d74de2926.png)

Reply:
I'm not against it's a new feature (as we didn't do that at the moment and this PR was just about adding some configuration on existing feature). Even if I'm pleased to do it, I will let it for another PR ;)

And, also, I see it as a distinct blame settings (why not initially set with this settings value) because we could like to display as relative in the application and the opposite in the blame (or the other way). Does it make sense?

Reply:
Yes, no worries.
Just flagging that we have an inconsistent UX.